public/app/features/dashboard-scene/pages/PublicDashboardScenePage.tsx

```diff
@@ -147,7 +147,9 @@ function getStyles(theme: GrafanaTheme2) {
     }),
     body: css({
       label: 'body',
+      display: 'flex',
       flex: 1,
+      flexDirection: 'column',
```

Comment:
might not even need this, just being extra safe about setting the flex direction since i've seen some browsers (*cough* safari *cough*) be extra picky about it